lib/index.js

```diff
@@ -6,7 +6,7 @@ import path from 'path';
 
 import handleData from './message';
 import type { Linter } from './types';
-import check from './helpers';
+import { check, findFlowPath } from './helpers';
```

Comment:
Once the check is the default export again, you can use:
`import check, {findFlowPath} from './helpers';`